app/next-client-app/app/(protected)/scanreports/[id]/downloads/columns.tsx

```diff
@@ -82,15 +84,32 @@ export const columns: ColumnDef<FileDownload>[] = [
     id: "Download",
     header: ({ column }) => <DataTableColumnHeader column={column} title="" />,
     cell: ({ row }) => {
-      const { id, scan_report } = row.original;
+      const { id, scan_report, file_type, name } = row.original;
+      const handleDownload = async () => {
+        const response = await downloadFile(scan_report, id, file_type.value);
```

Comment:
Can we call `downloadFiles` which I believe is server method from the client component?